src/llvm-late-gc-lowering.cpp

```diff
@@ -2218,14 +2218,14 @@ Value *LateLowerGCFrame::EmitTagPtr(IRBuilder<> &builder, Type *T, Type *T_size,
     assert(T == T_size || isa<PointerType>(T));
     auto TV = cast<PointerType>(V->getType());
     auto cast = builder.CreateBitCast(V, T->getPointerTo(TV->getAddressSpace()));
-    return builder.CreateInBoundsGEP(T, cast, ConstantInt::get(T_size, -1));
+    return builder.CreateInBoundsGEP(T, cast, ConstantInt::get(T_size, -1), "tag_ptr");
```

Comment:
Can we make this something like `V.getName() + ".tag"`, so that we propagate more naming information downward? (ditto for the tag load instruction).